app/lib/pre_assembly/digital_object.rb

```diff
@@ -142,14 +142,14 @@ def build_structural
                                          reading_order:)
       else
         build_from_staging_location(objects: object_files.sort,
-                                    content_metadata_creation: content_md_creation.to_sym,
+                                    processing_configuration:,
                                     content_md_creation_style:,
```

Comment:
unnecessary indirection of names removed, and `to_sym` done in the class initialize itself now)